pymc3/step_methods/sgmcmc.py

```diff
@@ -42,12 +42,13 @@ def elemwise_dlogL(vars, model, flat_view):
     # tensor of shape (batch_size,)
     logL = obs_var.logp_elemwiset.sum(axis=tuple(range(1, obs_var.logp_elemwiset.ndim)))
     # calculate fisher information
-    terms = tt.concatenate([\
-                theano.scan(lambda i, logX: theano.grad(logX[i], v).flatten(),\
+    terms = []
+    for var in vars:
```

Comment:
Also wrapping `lambda` function in theano might be problematic for multiple jobs I think.